docs/components/slider.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebarDepth: 0
+---
```

Comment:
please use documentation template to be consistent with other pages

Reply:
Updated doc page according to template with https://github.com/vuexp/vuexp/pull/14/commits/87816117337d13c9694b6716c379a40eb3f6f072

Updated doc page:
![image](https://user-images.githubusercontent.com/13813553/57077208-1fa15e80-6cf4-11e9-84be-bfedfa4df8cb.png)